introduction.md

```diff
@@ -8,4 +8,4 @@ Dojo Rules
 * My Rule1
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?